doc/governance.rst

```diff
@@ -0,0 +1,130 @@
+Voting on new core developers is one of the few activities that takes place on
+the project's private management list. While it is expected that most votes
+will be unanimous, a two-thirds majority of the cast votes is enough. The vote
+needs to be open for at least 1 week.
+
+Core developers that have not contributed to the project (commits or GitHub
+comments) in the past 12 months will be asked if they want to become emeritus
+core developers and recant their commit and voting rights until they become
+active again. The list of core developers, active and emeritus (with dates at
+which they became active) is public on the scikit-learn website.
+
+Technical Committee
+-------------------
+The Technical Committee (TC) members are core developers who have additional
+responsibilities to ensure the smooth running of the project. TC members are expected to
+participate in strategic planning, and approve changes to the governance model.
+The purpose of the TC is to ensure a smooth progress from the big-picture
+perspective. Indeed changes that impact the full project require a synthetic
+analysis and a consensus that is both explicit and informed. In cases that the
+core developer community (which includes the TC members) fails to reach such a
+consensus in the required time frame, the TC is the entity to resolve the
+issue.
+Membership of the TC is by nomination by a core developer. A nomination will
+result in discussion which cannot take more than a month and then a vote by
+the core developers which will stay open for a week. TC membership votes are
+subject to a two-third majority of all cast votes as well as a simple majority
+approval of all the current TC members. TC members who do not actively engage
+with the TC duties are expected to resign.
+ 
+The initial Technical Committee of scikit-learn consists of :user:`Alexandre Gramfort <agramfort>`,
+:user:`Olivier Grisel <ogrisel>`, :user:`Andreas Müller <amueller>`, :user:`Joel Nothman <jnothman>`,
+:user:`Hanmin Qin <qinhanmin2014>`, :user:`Gaël Varoquaux <GaelVaroquaux>`, and
+:user:`Roman Yurchak <rth>`.
+ 
+Decision Making Process
+=======================
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project contributors’ `mailing list <mailto:scikit-learn@python.org>`_
+and the `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_.
+Occasionally, sensitive discussion occurs on a private list.
+
+Scikit-learn uses a "consensus seeking" process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+At any point during the discussion, any core-developer can call for a vote, which will
+conclude one month from the call for the vote. Any vote must be backed by a
+`SLEP <slep>`. If no option can gather two thirds of the votes cast, the
```

Comment:
This phrasing is a bit ambiguous: are options referring to "yes" or "no?" Or is it referring to several technical options, and core developers don't agree with the same technical option?

Maybe I'm being to nitpicky here: I don't know how often this situation happens. We can always revisit if the TC is swamped with voting requests.

Reply:
Yes, it's ambiguous, I agree. I think right now it allows both (yes/no or different options). For things like freezing or pipeline slicing it might be less a question of whether we want it but how. We could have one SLEP and vote per technical option, I'm not sure how this would play out in practice. I think we can figure this out as we go along.